data/json/itemgroups/Weapons_Mods_Ammo/ammo.json

```diff
@@ -398,6 +404,9 @@
       { "item": "reloaded_762_51_incendiary", "prob": 5 },
       { "item": "reloaded_762_54R", "prob": 40 },
       { "item": "reloaded_762_m43", "prob": 20 },
+      { "item": "reloaded_303", "prob": 50 },
```

Comment:
.303 really should be  the rarest in this item group. it sees 0 use in modern military firearms, 303 rifles can be easily converted to fire 308/7.62x51 and export versions were frequently converted, surplus military ammo usually has berdan primers which complicates using the casings for reloading (even modern ammo sometimes has this). 

it should be in roughly the same itemgroups as 7.62x54r but a lot more rare since .303 doesnt get used in any modern military conflicts.